rfcs/023-images-endpoint/README.md

```diff
@@ -0,0 +1,48 @@
+# RFC 023: Images endpoint
+
+**Status:** :construction: Draft :building_construction:
+
+**Last updated:** 27/04/2020
+
+## Context and requirements
+
+- Images are now entities in the catalogue API
+- We want to be able to search for images, and to fetch specified images
+- The images endpoint should behave similarly to the works endpoint
+
+## Proposal
+
+The initial images endpoint will be in **beta**, and therefore will be liable to have breaking changes made without a version bump.
+
+### Image display model
+
+```
+{
+  "id": String,
+  "location": DigitalLocation,
+  "parentWork": String
+}
+```
+
+This is intentionally minimal at this point as we decide what fields an image should have.
```

Comment:
So on the assumption that for search purposes, we're copying all relevant data into a hidden full text field in the index and this is just for display. I'd say it's what is needed to render the panel on search results, because otherwise you'll need to make another API request for that data, which seems suboptimal.